release-notes/opensearch-release-notes-2.13.0.md

```diff
@@ -0,0 +1,796 @@
+<li>[Backport main] Add release notes for 1.3.15 by @opensearch-trigger-bot in https://github.com/opensearch-project/sql/pull/2538</li>
+<li>Upgrade opensearch-spark jars to 0.3.0 by @noCharger in https://github.com/opensearch-project/sql/pull/2568</li>
+</ul>
+
+<h3>Dashboards Assistant</h3>
+
+<ul>
+<li>Add new workflow to verify binary install. (<a href="https://github.com/opensearch-project/dashboards-assistant/pull/159">#159</a>)</li>
+</ul>
+
+<h3>Dashboards Observability</h3>
+
+<ul>
+<li>Add single version flag during bootstrap to fix version conflicts (<a href="https://github.com/opensearch-project/dashboards-observability/pull/1460">#1460</a>)</li>
+</ul>
+
+<h3>OpenSearch Dashboards Reporting</h3>
+
+<ul>
+<li>Add single version flag during bootstrap to fix version conflicts(<a href="https://github.com/opensearch-project/dashboards-reporting/pull/303">#303</a>)</li>
+<li>Add workflow to verify binary installation is successful  (<a href="https://github.com/opensearch-project/dashboards-reporting/pull/313">#313</a>)</li>
+</ul>
+
+<h3>OpenSearch Dashboards Search Relevance</h3>
+
+<ul>
+<li>Add workflow to verify binary installation (<a href="https://github.com/opensearch-project/dashboards-search-relevance/pull/373">#373</a>) (<a href="https://github.com/opensearch-project/dashboards-search-relevance/pull/375">#375</a>)</li>
+</ul>
+
+<h3>OpenSearch Dashboards Visualizations</h3>
+
+<ul>
+<li>Add workflow to verify binary installation (<a href="https://github.com/opensearch-project/dashboards-visualizations/pull/347">#347</a>)</li>
+</ul>
+<ul>
+<li>[CI/CD] Add single version flag during bootstrap to fix version conflicts (<a href="https://github.com/opensearch-project/dashboards-visualizations/pull/341">#341</a>)</li>
+</ul>
+
+<h3>OpenSearch ML Commonss Dashboards</h3>
+
+<ul>
+<li>Add new workflow to verify binary install. (<a href="https://github.com/opensearch-project/ml-commons-dashboards/pull/306">#306</a>)</li>
+</ul>
+
+<h3>OpenSearch Query Workbench</h3>
+
+<ul>
+<li>Add single version flag during bootstrap to fix version conflicts (<a href="https://github.com/opensearch-project/dashboards-query-workbench/pull/269">#269</a>)</li>
+</ul>
+<ul>
+<li>Add OSD react as required plugin (<a href="https://github.com/opensearch-project/dashboards-query-workbench/pull/278">#278</a>)</li>
+</ul>
+
+<h3>OpenSearch Security Analytics Dashboards</h3>
+
+<ul>
+<li>Updated cypress version to match core OSD (<a href="https://github.com/opensearch-project/security-analytics-dashboards-plugin/pull/907">#907</a>)</li>
+</ul>
+
+<h3> OpenSearch Build </h3>
+
+<ul>
+<li>Fix default config permissions for RPM and Debian Packages (<a href="https://github.com/opensearch-project/opensearch-build/issues/3815">#3815</a>)</li>
+<li>Fix OpenSearch and OpenSearch-Dashboards Service restart after upgrading the package (<a href="https://github.com/opensearch-project/opensearch-build/issues/3891">#3891</a>)</li>
```

Comment:
`Allow automatically restarting of OpenSearch and OpenSearch-Dashboards Service after upgrading the Debian package`

Use this PR: https://github.com/opensearch-project/opensearch-build/pull/4530